systemjs/react-hello-world/src/react-hello-world.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+ReactDOM.render(<button>A button created by React</button>, document.getElementById('react-root'));
```

Comment:
Still might be interesting to make this example multi-module.

Reply:
I'm open to that -- maybe let's do that in a future PR